Automattic/studio
# Studio: Update "Create a free account" link to redirect to magic signup flow

Fixes 5531-gh-Automattic/dotcom-forge and 5532-gh-Automattic/dotcom-forge

## Proposed Changes

- Following https://github.com/Automattic/wp-calypso/pull/90204, users who choose to create an account from the WordPress.com login page will be redirect to the magic signup flow.
- With this PR, we update the "Create a free account" link in the Studio app to redirect to the new signup flow.

## Testing Instructions

* Log out of both WordPress.com and the Studio app.
* In the Studio app, navigate to the **Share** tab.
* Click on the "Create a free account" link in the Studio app.
* Verify you're taken to the magic signup page and enter an email that's not already associated with a WordPress.com account.
* Go through the flow to complete account setup and verify you're automatically logged into the Studio app.  

| Link  | Form |
| ------------- | ------------- |
| <img src="https://github.com/Automattic/studio/assets/2998162/11e6ad6d-8e27-4718-aa33-d8c0642d11e0" width="100%"> | <img src="https://github.com/Automattic/studio/assets/2998162/eadb7054-88fe-4103-8abb-b4bc2675eed6" width="100%"> |

## Pre-merge Checklist
